tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
-    "module": "CommonJS",
+    "module": "NodeNext",
+    // "moduleResolution":"node",
+    "target": "ES6",
     "baseUrl": "src",
-    "outDir": "./dist",
-    "esModuleInterop": true,
+    "outDir": "dist",
     "sourceMap": true,
-    "noImplicitAny": true,
-    "allowJs": true
+    "noImplicitAny": true
```

Comment:
why did you change this file?

Reply:
It was due to the errors I had before, but now I have corrected it